perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -3317,7 +3317,7 @@ public void removeAllAttributes(PerunSession sess, Resource resource) throws Int
 	public boolean removeAttribute(PerunSession sess, Resource resource, Member member, AttributeDefinition attribute) throws InternalErrorException {
 		try {
 			if (0 < jdbc.update("DELETE FROM member_resource_attr_values WHERE attr_id=? AND member_id=? AND resource_id=?", attribute.getId(), member.getId(), resource.getId())) {
-				log.info("Attribute (its value) was removed from member on resource. Attribute={}, member={}, resource=" + resource, attribute, member);
+				log.debug("Attribute value {} was removed from member {} on resource {}" + resource, attribute, member);
```

Comment:
Technically its usefull to have full objects in the message, but also in this case we can't relay on previous (correct) value to be present in passed ``AttributeDefinition`` (which, in a fact, lacks value). So maybe rephrasing all message to something like this would help: ``Attribute value for {} was removed from member {} on resource {}.`` where you again pass attribute name and then member and resource objects.
Also there is an inconsistency with ending dot between this kind of messages.